In [2]:
import pandas as pd
pd.set_option('display.max_columns', None)

In [3]:
population = pd.read_csv('total-population.csv')

print ("Estimated Total Population:",  population['Estimate; Total'].sum(axis=0))
print("")

sortedP = population.sort_values('Estimate; Total')
print (sortedP.head(3))
print ("")
print (sortedP.tail(3))

Estimated Total Population: 309327.0

   Neighborhood  Id  Estimate; Total  Margin of Error; Total
21      Chateau  22              3.0                3.000000
71  South Shore  72             12.0               14.000000
57  North Shore  58            205.0               66.753277

           Neighborhood  Id  Estimate; Total  Margin of Error; Total
13            Brookline  14          13160.0              943.065215
68            Shadyside  69          13562.0              861.984919
77  Squirrel Hill South  78          16042.0              889.712313


## Estimated Total Population: 309327 People

# The 3 most populated neighborhoods:

- Squirrel Hill South: 16042 People
- Shadyside: 13562 People
- Brookline: 13160 People 

# The 3 least populated neighborhoods:

- North Shore: 205 People
- South Shore: 12 People
- Chateau: 3 People

# Looking at Crime data compared to Population data:

In [28]:
#Crime Data
crime = pd.read_csv('crime.csv')

#Sorted Alphabetically
crime = crime.sort_values('INCIDENTNEIGHBORHOOD')['INCIDENTNEIGHBORHOOD'].value_counts(sort = False).rename_axis("Neighborhood")
print(crime.to_string())

Neighborhood
Allegheny Center              26
Allegheny West                 6
Allentown                     44
Arlington                     23
Arlington Heights              6
Banksville                    16
Bedford Dwellings             29
Beechview                     66
Beltzhoover                   15
Bloomfield                    52
Bluff                         51
Bon Air                       16
Brighton Heights              73
Brookline                     76
California-Kirkbride          17
Carrick                      118
Central Business District    210
Central Lawrenceville         29
Central North Side            28
Central Oakland               36
Chateau                       31
Crafton Heights               45
Crawford-Roberts              48
Duquesne Heights              14
East Allegheny                56
East Hills                    67
East Liberty                  91
Elliott                       24
Esplen                         3
Fairywood                     

# The number of crimes committed in the 3 most populated neighborhoods:

- Squirrel Hill South: 51 Cases
- Shadyside: 54 Cases
- Brookline: 76 Cases

# The number of crimes committed in the 3 least populated neighborhoods:

- North Shore: 23 Cases
- South Shore: 13 Cases
- Chateau: 31 Cases [^1]

[^1]: There seems to be a mistake with the population data or Crime data regarding Chateau

# Looking at Housing Prices Compared to Population Data

In [4]:
value = pd.read_csv('value.csv')
print(value.loc[value['Neighborhood'] == "Squirrel Hill South"])
print("XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
print(value.loc[value['Neighborhood'] == "Shadyside"])
print("XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
print(value.loc[value['Neighborhood'] == "Brookline"])
print("XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
print(value.loc[value['Neighborhood'] == "North Shore"])
print("XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
print(value.loc[value['Neighborhood'] == "South Shore"])
print("XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
print(value.loc[value['Neighborhood'] == "Chateau"])

Neighborhood  Id  Estimate; Total:  Margin of Error; Total:  \
77  Squirrel Hill South  78            3069.0                245.10406   

    Estimate; Total: - Less than $10,000  \
77                                  38.0   

    Margin of Error; Total: - Less than $10,000  \
77                                    47.916594   

    Estimate; Total: - $10,000 to $14,999  \
77                                    0.0   

    Margin of Error; Total: - $10,000 to $14,999  \
77                                     36.055513   

    Estimate; Total: - $15,000 to $19,999  \
77                                   32.0   

    Margin of Error; Total: - $15,000 to $19,999  \
77                                     50.803543   

    Estimate; Total: - $20,000 to $24,999  \
77                                    0.0   

    Margin of Error; Total: - $20,000 to $24,999  \
77                                     36.055513   

    Estimate; Total: - $25,000 to $29,999  \
77                                   11.0   

    Margin of Error; Total: - $25,000 to $29,999  \
77                                     35.454196   

    Estimate; Total: - $30,000 to $34,999  \
77                                    0.0   

    Margin of Error; Total: - $30,000 to $34,999  \
77                                     36.055513   

    Estimate; Total: - $35,000 to $39,999  \
77                                   16.0   

    Margin of Error; Total: - $35,000 to $39,999  \
77                                     42.720019   

    Estimate; Total: - $40,000 to $49,999  \
77                                    0.0   

    Margin of Error; Total: - $40,000 to $49,999  \
77                                     36.055513   

    Estimate; Total: - $50,000 to $59,999  \
77                                   15.0   

    Margin of Error; Total: - $50,000 to $59,999  \
77                                     41.581246   

    Estimate; Total: - $60,000 to $69,999  \
77                                    0.0   

    Margin of Error; Total: - $60,000 to $69,999  \
77                                     36.055513   

    Estimate; Total: - $70,000 to $79,999  \
77                                   24.0   

    Margin of Error; Total: - $70,000 to $79,999  \
77                                     44.158804   

    Estimate; Total: - $80,000 to $89,999  \
77                                   30.0   

    Margin of Error; Total: - $80,000 to $89,999  \
77                                     42.154478   

    Estimate; Total: - $90,000 to $99,999  \
77                                   23.0   

    Margin of Error; Total: - $90,000 to $99,999  \
77                                     43.600459   

    Estimate; Total: - $100,000 to $124,999  \
77                                    220.0   

    Margin of Error; Total: - $100,000 to $124,999  \
77                                       96.824584   

    Estimate; Total: - $125,000 to $149,999  \
77                                    137.0   

    Margin of Error; Total: - $125,000 to $149,999  \
77                                       81.277303   

    Estimate; Total: - $150,000 to $174,999  \
77                                    196.0   

    Margin of Error; Total: - $150,000 to $174,999  \
77                                       78.974679   

    Estimate; Total: - $175,000 to $199,999  \
77                                    184.0   

    Margin of Error; Total: - $175,000 to $199,999  \
77                                        78.25599   

    Estimate; Total: - $200,000 to $249,999  \
77                                    457.0   

    Margin of Error; Total: - $200,000 to $249,999  \
77                                      137.338997   

    Estimate; Total: - $250,000 to $299,999  \
77                                    322.0   

    Margin of Error; Total: - $250,000 to $299,999  \
77                                      115.351636   

    Estimate; Total: - $300,000 to $399,999  \
77                                    593.0   


XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
   Neighborhood  Id  Estimate; Total:  Margin of Error; Total:  \
68    Shadyside  69            2030.0               225.794154   

    Estimate; Total: - Less than $10,000  \
68                                   0.0   

    Margin of Error; Total: - Less than $10,000  \
68                                    37.416574   

    Estimate; Total: - $10,000 to $14,999  \
68                                   11.0   

    Margin of Error; Total: - $10,000 to $14,999  \
68                                     40.298883   

    Estimate; Total: - $15,000 to $19,999  \
68                                    0.0   

    Margin of Error; Total: - $15,000 to $19,999  \
68                                     37.416574   

    Estimate; Total: - $20,000 to $24,999  \
68                                    9.0   

    Margin of Error; Total: - $20,000 to $24,999  \
68                                     3

# Showing the Price Ranges that have 100+ Houses of the Most Populated Areas

* Squirrel Hill South: 100k-999.9k
* Shadyside: 100k-124.9k | 200k-749.9k
* Brookline: 40k-174.9k

# Showing the Price Ranges that have 100+ Houses of the Least Populated Areas

* North Shore: N/A (Listed, but all values are 0)
* South Shore: N/A (Listed, but all values are 0)
* Chateau: N/A (Listed, but all values are 0)
* No houses in these areas? 